armadillo/src/main/java/org/molgenis/armadillo/profile/DockerService.java

```diff
@@ -74,12 +81,47 @@ public Map<String, ContainerInfo> getAllProfileStatuses() {
     return statuses;
   }
 
+  // `docker container ps` show these name structure
+
+  /**
+   * The container can run in its own network/compose.
+   *
+   * <p>Both the profiles and/or Armadillo can be part of a docker-compose.yml. You can check with
+   * `docker container ps` to see this name structure.
+   *
+   * @param profileName the profile name set by DataManager.
+   * @return adjusted container name if applicable.
+   */
+  String asContainerName(String profileName) {
+    if (!inContainer) {
+      LOG.warn("NO ".repeat(100) + " " + profileName);
```

Comment:
Do we need this information: not a container, in a container, with prefix?

If yes, rephrase this NO is more useful information. Same goes for the other LOG within same function